hnn_core/cell_response.py

```diff
@@ -322,14 +322,15 @@ def mean_rates(self, tstart, tstop, gid_ranges, mean_type='all'):
 
         return spike_rates
 
-    def plot_spikes_raster(self, ax=None, show=True):
+    def plot_spikes_raster(self, trial_idx=None, ax=None, show=True):
```

Comment:
this is an API breaking change. It will break code which does:

```py
plot_spikes_raster(ax)
```

as a general rule, you want to put new arguments towards the end, but maybe before `show` so API is consistent. Anyhow, I don't think we are doing deprecations at this point, so I'll let this pass but something to remember for the future :)

Reply:
Good catch, might be worth discussing the mechanics of deprecation warnings at some point. Especially identifying functions that are the most likely to change in the near future